google-drive.html

```diff
@@ -158,7 +158,7 @@
           onComplete: function (data) {
             this._uploadedList.push(JSON.parse(data));
             this.fire('google-drive-upload-status', { status: 'Upload successful' });
-            this._uploadList = [];
+            this.splice('_uploadList', 0, this._uploadList.length);
```

Comment:
TIL `set`

is it not on the doc page on purpose?
https://www.polymer-project.org/1.0/docs/devguide/properties.html#array-mutation